mlir/lib/Conversion/TosaToTensor/TosaToTensorPass.cpp

```diff
@@ -42,7 +43,10 @@ struct TosaToTensor : public impl::TosaToTensorBase<TosaToTensor> {
     target.addLegalDialect<arith::ArithDialect>();
     target.addLegalDialect<tensor::TensorDialect>();
 
-    mlir::tosa::populateTosaToTensorConversionPatterns(&patterns);
+    TypeConverter converter;
+    mlir::tosa::populateTosaToLinalgTypeConversion(converter);
```

Comment:
Doesn't this make the TosaToTensor specifically targeted towards Linalg ops? I don't know of any other possible lowerings but still... I'd like to make sure.

Reply:
It uses the type converter from there, but it still only generate `tensor` ops. I don't know of a better place to put the type converter; maybe upstream will have an idea.